boards/arduino_mkrzero/Cargo.toml

```diff
@@ -15,18 +15,24 @@ version = "0.7"
 optional = true
 
 [dependencies.atsamd-hal]
+path = "../../hal"
```

Comment:
This line should be removed. `arduino_mkrzero` is a Tier 2 board, so it should use a `crates.io` dependency to `atsamd-hal` rather than a path dependency.

Reply:
If I remove that line, the build fails with
```
the trait `Monotonic` is not implemented for `arduino_mkrzero::rtc::Rtc<arduino_mkrzero::rtc::Count32Mode>`
```
Does that mean upstream `atsamd-hal` needs to catch up? I don't quite understand what the issue is but if it's a matter of implementing `Monotonic`, I'd be happy to give it a try.

Reply:
That's because the latest `crates.io` `atsamd-hal` version still has one of the `0.6.0-rc.x` RTIC dependencies. `Monotonic` is implemented, but there's a version mismatch with you `cortex-m-rtic = "1.0"` dependency. You can either wait for atsamd-hal to catch up, which should hopefully happen soon, or match your RTIC version with the one from `atsamd-hal`.

Reply:
Alright, let's give atsamd-hal some time and wait. Thanks for the insight.